libagbsyscall/libagbsyscall.s

```diff
@@ -1,9 +1,12 @@
 	.include "../constants/gba_constants.inc"
 	.include "../asm/macros/function.inc"
+	
+	.syntax unified
 
 	.text
 
 	.set SOFT_RESET_DIRECT_BUF, 0x03007FFA
+	.set USER_STACK,			0x03007F00
```

Comment:
Alignment

Reply:
they are aligned though. one with tabs, the other with spaces?

Reply:
There's a big 3 tab gap here for no reason, either align it with `0x03007FFA` with spaces or just use a single space